examples/combined_ODNP.py

```diff
@@ -77,7 +79,7 @@
 powers = 1e-3 * 10 ** (dB_settings / 10.0)
 # }}}
 # {{{ these change if we change the way the data is saved
-IR_postproc = "spincore_IR_v3" # note that you have changed the way the data is saved, and so this should change likewise!!!!
+IR_postproc = "spincore_IR_v1" # note that you have changed the way the data is saved, and so this should change likewise!!!!
```

Comment:
why are you rolling this back? see comment

Reply:
you reply to this in a different comment, and I note that the change seems to be being OK with no phase cycle -- is the postproc OK with that

Reply:
as of right now there is no proc_script for an IR that is not phase cycled. There is one that assists a 2 x 2 phase cycle (v1) and one for a 4 x 4 (v2)

Reply:
so yes we can make it v3